driver-core/src/main/java/com/datastax/driver/core/ProtocolOptions.java

```diff
@@ -231,4 +232,23 @@ public AuthProvider getAuthProvider() {
         return authProvider;
     }
 
+    /**
+     * Allows client to connect to server using latest development protocol version,
+     * which is currently in beta.
+     *
+     * @param useBeta
+     * @return this {@code ProtocolOptions} object.
+     */
+    public ProtocolOptions setAllowBetaProtocolVersions(boolean useBeta) {
```

Comment:
true, I've only added it here because I wanted to reduce the amount of constructors. But you're right, it's better to do it here.